manifests/init.pp

```diff
@@ -320,6 +320,12 @@
 #   Whether we should test new configuration files with `corosync -t`.
 #   (requires corosync 2.3.4)
 #
+# @param major_version_corosync
```

Comment:
please also run `bundle exec rake reference` to update the REFERENCE.md (it uses those comments to generate markdown documentation).